config/locales/nl.yml

```diff
@@ -183,6 +184,7 @@ nl:
       update_failed_msg: Deze emoji kon niet worden bijgewerkt
       updated_msg: Bijwerken van emoji is geslaagd!
       upload: Uploaden
+      by_domain: Domein
```

Comment:
Please move this line below [line 163](https://github.com/jeroenpraat/mastodon/blob/cb5527e73002c60fda85514052a0e2c2d9ed68cc/config/locales/nl.yml#L163)